guides/common/modules/proc_registering-hosts-for-conversion.adoc

```diff
@@ -0,0 +1,23 @@
+[id="registering-hosts-for-conversion_{context}"]
+
+= Registering Hosts for Conversion
+
+Use this procedure to register the hosts to {Project}.
+
+.Procedure
+. In the {ProjectWebUI}, navigate to *Hosts* *>* *All Hosts* and click *Register Host*.
+. In the *Host Group* field, select the Host Group you want to convert.
```

Comment:
we're not converting Host Groups, perhaps this formulation would be better

select the Host Group you want to use as a template for the registration

Reply:
Fixed.